models/product-type/src/types.ts

```diff
@@ -0,0 +1,22 @@
+import { ProductType, ProductTypeDraft } from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TProductType = ProductType;
+
+export type TProductTypeDraft = ProductTypeDraft;
+
+export type TProductTypeGraphql = TProductType & {
+  __typename: 'ProductType';
```

Comment:
Looks like the `typename` is `ProductTypeDefinition`
![image](https://user-images.githubusercontent.com/8136478/218781428-1deda5e6-7514-4261-b125-7a23b4ac7638.png)

Reply:
🔧  Fixed with [2f4ca81](https://github.com/commercetools/test-data/pull/198/commits/2f4ca8174a0c9595ff0eb70101921a68468f89be)